app/templates/user/certificates-view.jade

```diff
@@ -78,6 +78,10 @@ style.
               span(data-i18n="courses.teacher")
               span= ': '
               span= view.teacherName
+          if view.serialNumber
+            .serial-number-info
+              span= 'No. '
```

Comment:
We probably want to translate this. Even if we might happen to want "No. 267278" in Chinese also, other languages will want to localize. Can we add a translation prefix for this?

Reply:
How can I add a constant for all the locale files in the same time?